### PR TITLE
Improve comments in example

### PR DESCRIPTION
**Pull request by C0D3O**

### CODE REVIEW

File: examples/single-price.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ offchainOracle.getRateToEth(
         const numerator = 10 ** token.decimals;
         const denominator = 1e18; // eth decimals
         const price = parseFloat(rate) * numerator / denominator / 1e18;
-        console.log(price.toString()); // USDT in ETH
```
Comment (zZoMROT) [inline examples/single-price.js:24]: This comment implies the value of USDT in the currency WETH, and in this case, the comments match what is output to the console, as price reflects how much WETH is needed for 1 USDT. Due to this misunderstanding, your comment may also be considered incorrect after some time. It seems worth expanding comments further for clarity.

Comment (C0D3O) [reply examples/single-price.js:24]: So you mean that it doesn't show how much ETH is in USDT? But the.... What else can it show?)) 
Your code console.log says 3800 eth in USDT and 0.000253747 USDT in eth. Isn't it mixed up?

Comment (C0D3O) [reply examples/single-price.js:24]: Or the comment means 1 eth is USDT 
And 1 USDT in eth?

File: examples/single-price.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ offchainOracle.getRateToEth(
...
-        console.log((1 / price).toString()); // ETH in USDT
+        console.log(price.toString()); // ETH in USDT
+        console.log((1 / price).toString()); // USDT in ETH
```
Comment (zZoMROT) [inline examples/single-price.js:25]:
```suggestion
        console.log(price.toString()); // how much ETH needed for 1 USDT
        console.log((1 / price).toString()); // how much USDT needed for 1 ETH
```

File: examples/single-price.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ offchainOracle.getRateToEth(
         const numerator = 10 ** token.decimals;
         const denominator = 1e18; // eth decimals
```
Comment (zZoMROT) [inline examples/single-price.js:22]:
```suggestion
        const denominator = 1e18; // ETH decimals
```

**Review by zZoMROT — APPROVED** [review]

**Pull request merged**